genesyscloud/resource_genesyscloud_webdeployments_configuration_test.go

```diff
@@ -86,6 +88,11 @@ func TestAccResourceWebDeploymentsConfigurationComplex(t *testing.T) {
 					resource.TestCheckResourceAttr(fullResourceName, "messenger.0.file_upload.0.mode.1.file_types.#", "1"),
 					resource.TestCheckResourceAttr(fullResourceName, "messenger.0.file_upload.0.mode.1.file_types.0", "image/jpeg"),
 					resource.TestCheckResourceAttr(fullResourceName, "messenger.0.file_upload.0.mode.1.max_file_size_kb", "123"),
+					resource.TestCheckResourceAttr(fullResourceName, "cobrowse.#", "1"),
+					resource.TestCheckResourceAttr(fullResourceName, "cobrowse.0.enabled", "true"),
```

Comment:
Small thing: we usually use the variable `trueValue` for the string "true". Same for `falseValue` and `nullValue`.